doc/guides/porting/board_porting.rst

```diff
@@ -503,6 +508,201 @@ including ``nrfjprog.board.cmake`` first means that ``nrjfprog`` is the default
 flash runner for this board. Since ``nrfjprog`` does not support debugging,
 ``jlink`` is the default debug runner.
 
+.. _porting_board_revisions:
+
+Multiple board revisions
+************************
+
+See :ref:`application_board_version` for basics on this feature from the user
+perspective.
+
+To create a new board revision for the ``plank`` board, create these additional
+files in the board folder:
+
+.. code-block:: none
+
+   boards/<ARCH>/plank
+   ├── plank_<revision>.conf
+   ├── plank_<revision>.overlay  # optional
+   └── revision.cmake
+
+When the user builds for board ``plank@<revision>``:
+
+- Additional Kconfig settings specified in the file
+  :file:`plank_<revision>.conf` will be merged into the board's default Kconfig
+  configuration.
+
+- The optional devicetree overlay :file:`plank_<revision>.overlay` will be added
+  to the common :file:`plank.dts` devicetree file
+
+- The :file:`revision.cmake` file controls how the Zephyr build system matches
+  the ``<board>@<revision>`` string specified by the user when building an
+  application for the board.
+
+.. important::
+
+   If you only need a devicetree overlay, you must still create an empty
+   :file:`<board>_<revision>.conf` file.
+
+Currently, ``<revision>`` can be either a numeric ``MAJOR.MINOR.PATCH`` style
+revision like ``1.5.0``, or single letter like ``A``, ``B``, etc. Zephyr
+provides a CMake board extension function, ``board_check_revision()``, to make
+it easy to match either style from :file:`revision.cmake`.
+
+The following sections describe how to support these styles of revision
+numbers.
+
+Numeric revisions
+=================
+
+Let's say you want to add support for revisions ``0.5.0``, ``1.0.0``, and
+``1.5.0`` of the ``plank`` board. Create :file:`revision.cmake` with the above
```

Comment:
Not clear what the "above content" refers to here.

Reply:
fixed. some left-over from reworking the docs.
Thanks for catching it.